src/CSI.IBTA.Employer/Clients/EmployeesClient.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CSI.IBTA.Employer.Endpoints;
+using CSI.IBTA.Employer.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using Newtonsoft.Json;
+
+namespace CSI.IBTA.Employer.Clients
+{
+    public class EmployeesClient : IEmployeeClient
```

Comment:
Make naming consistent. if class is in plural Employee**s**, the i-face should be plural too IEmployee**s**